Composer/packages/webchat-client/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
I don't think this deserves it's own package. Each package added adds time to our build pipeline. We are getting to the point now that that is becoming a burden.

Reply:
Moved it to Composer client now